### DISCUSSION TITLE
No Fullscreen button showing in iPhone

### DISCUSSION THREAD

**Discussion by gab-logicbase** [igordanchenko/yet-another-react-lightbox#189] [Q&A] [answered]
![image](https://github.com/igordanchenko/yet-another-react-lightbox/assets/116131283/7240d525-9321-418e-958c-601b00affdc0)

The following toolbar is not showing in iphone device but on android it's working fine. Is there any configuration to be enabled for ios?

**Comment by igordanchenko** [comment, marked as answer]
Please see the very first paragraph in the Fullscreen plugin documentation - https://yet-another-react-lightbox.com/plugins/fullscreen

**Reply by gab-logicbase** [reply]
Is there a way we could support this in the future? Or any alternative you could provide to override it?

**Reply by igordanchenko** [reply]
> Is there a way we could support this in the future? 

Sure, as soon as Apple implements support for [Fullscreen API](https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API#browser_compatibility) on iPhone.

**Reply by gab-logicbase** [reply]
Since Fullscreen API is supported for version 16.4, how can I help to integrate this one?

**Reply by igordanchenko** [reply]
You can't as the Fullscreen API is still not supported on iPhone. If you pay attention to the browser compatibility table that I referenced above, you can see that it clearly says "Only available on iPad, not on iPhone"
